scripts/migrate_unconfirmed_valid_users.py

```diff
@@ -0,0 +1,91 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+"""Script to migrate users with a valid date_last_login but no date_confirmed."""
+
+import sys
+import logging
+
+from website.app import init_app
+from website.models import User
+from scripts import utils as script_utils
+from tests.base import OsfTestCase
+from tests.factories import UserFactory
+from modularodm import Q
+import datetime as dt
+
+logger = logging.getLogger(__name__)
+INFO_MSG = 'Migrating user'
+
+def do_migration(records):
+    for user in records:
+        log_info(user)
+        user.date_confirmed = user.date_last_login
+        if not user.is_registered:
+            user.is_registered = True
+        user.save()
+    logger.info('Migrated {0} users'.format(len(records)))
+        
+
+def get_targets():
+    return User.find(Q('date_confirmed', 'eq', None) & Q('date_last_login', 'ne', None))
+
+
+def log_info(user):
+    logger.info('{0}- {1}: date_confirmed={2}, \
```

Comment:
No need to use `\' for line continuations (see [here](http://cosdev.readthedocs.org/en/latest/style_guides/python.html#line-lengths).